artemis-server/src/main/java/org/apache/activemq/artemis/core/postoffice/impl/WildcardAddressManager.java

```diff
@@ -30,6 +25,11 @@
 import org.apache.activemq.artemis.core.postoffice.BindingsFactory;
 import org.apache.activemq.artemis.core.transaction.Transaction;
 
+import java.util.Collection;
```

Comment:
This move re-change wasn't needed, if could revert, so just the code changes needed are in the PR.